### ISSUE TITLE
Duplicate timestamps in .csv output

### ISSUE THREAD

**Issue by aklohr** [movestore/Daily-Rest-Sites#3]
This app is generating duplicate timestamps (duplicate rows), which is causing errors in downstream apps (e.g. Write Shapefile) due to timestamps being out of order. Not sure why timestamps are repeated in this app output...
![image](https://github.com/user-attachments/assets/0fe0f68e-27ad-4e55-9cba-0a83f0a50895)

Workflow "8. Roost and foraging site extraction" shared with Anne for troubleshooting. Also see [Issue #13](https://github.com/movestore/write-shapefile/issues/13) in Write Shapefile.

**Comment by andreakoelzsch** [comment]
Hi @aklohr, Anne has found an error and I have just built and included to moveApps the new version of the App. Please can you test if it works correctly for you now? Thanks!

**Comment by aklohr** [comment]
Hi @andreakoelzsch , thanks for pushing a new version! And thanks to Anne for troubleshooting. The workflow runs now (the shapefile app creates output without errors). I checked the .csv output from the Daily Rest/Foraging App and at a quick glance I didn't see any duplicate timestamps.

I did notice the columns in the .csv output seem to have been re-ordered. Now the first ~20 columns are accessory measurements, meaning I have to scroll pretty far over to see timestamps & lat/long. Is it possible to reorder the attributes so the "primary" ones (i.e.., individual_local_identifier, timestamp, lat/long) come first, or can that not be adjusted?
